### ISSUE TITLE
Authentication Failure despite correct BPI Key

### ISSUE THREAD

**Issue by qmilangowin** [quantnetwork/overledger-network-gateway#5]
In attempting to launch the OVN Gateway, I get an authentication issue now, despite using a BPI key that worked in the past. So not sure if this is a downgrade? The BPI Key is for the test net.

Attaching logs
[ovn_logs.log](https://github.com/quantnetwork/overledger-network-gateway/files/6163398/ovn_logs.log)

**Comment by TigerMike2004** [comment]
They have a 1.5 and 2.0! Wonder  if that has anything to do with it?

**Comment by TigerMike2004** [comment]
I just got on Git Hub and seen it! Let me get you a email to someone on
Quant team to help you!

**Comment by qmilangowin** [comment]
Ok great thanks. I know it worked in the past with the BPI key I have. I've even regenerated it and same issue.  So whether it's an issue on my side or on the side of the latest revision for this, thought I'd submit the issue.

**Comment by TigerMike2004** [comment]
Do you have Quant  on Telegram  or Twitter! Or contact Quant! Google it and
add it to homescreen

**Comment by qmilangowin** [comment]
Yes I've asked this a while back in the Dev channel on Telegram without any resolution.

**Comment by TigerMike2004** [comment]
Google it and add Quant  on homescreen! They have people that can solve it
easily

**Comment by TigerMike2004** [comment]
Use Chrome  though!

**Comment by qmilangowin** [comment]
Not sure what adding Quant to my google homescreen is supposed to accomplish apart from making their website my start-up page. That's not the issue here. 
Given that this is a open repo, I'm submitting what I believe might be an issue with latest build. That's how git is used, if using it for issue submissions.

**Comment by TigerMike2004** [comment]
Yeah I was only trying to get you with  Quant! Telegram  has a Quant  page
of development

**Comment by TigerMike2004** [comment]
This is your help

**Comment by qmilangowin** [comment]
I'll take my chances that someone from the QNT team will analyse my issue here as this is the proper medium for such exchanges. I'm not a fan of telegram, I work in development, I use github for such things. Thanks.

**Comment by hambudge** [comment]
I did an upgrade just now (I needed to reboot remote server to stop and remove old container-second time I had to do that this year). I'm running the curl command just now to test my gateway but showing "connection refused"

**Comment by dickcheney** [comment]
Was anybody able to get resolution on this ? I am having the same issue.
